test/services/activatable/activation_service_test.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+module Activatable
+  class ActivationServiceTest < ActiveSupport::TestCase
+    setup do
+      prospective = prospective_students(:mine)
+
+      @activation =
+        Activatable::ActivationService.new(
+          id_number: prospective.id_number,
+          first_name: prospective.first_name,
+          last_name: prospective.last_name,
+          date_of_birth: '1984-11-16',
+          serial: 'J10',
+          serial_no: '94646',
+          mobile_phone: '5551111111',
+          country: 'TR'
+        )
+    end
+
+    # validations: presence
+    %i[
```

Comment:
Testlerle ilgili helper yazmıştık.  Kontrol etmeden yazıyorum, buna uygun şekilde değiştirilmesi gerekebilir. https://github.com/omu/nokul/blob/develop/doc/development/test-unit.md  Eğer böyle bir durum varsa tüm testleri gözden geçirmende yarar var.

Reply:
Hocam activationservice sınıfının activerecord modeli olarak bir karşılığı olmadığı için bu helperları kullanamıyorum.